### PR TITLE
feat(TDP-4797): Implement rerun failed Test on Times Component

### PR DESCRIPTION
**Pull request by Avi247h**
### Description

Please include a summary of the change along with relevant motivation and context.

https://nidigitalsolutions.jira.com/browse/TDP-4797

### Checklist

- [ ] Have you done any manual testing?
- [ ] Does it have automated tests?
- [ ] Do you need any other PRs merged before this (if so please list)?
- [ ] Do you need to update the README/Runbook
- [ ] Have you checked for [accessibility](https://nidigitalsolutions.jira.com/wiki/spaces/TNLDigital/pages/3898048523/Accessibility+Dev+Guide#Checklist)?

### Screenshots (if appropriate):

Include screenshots if needed.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -106,7 +107,7 @@ jobs:
       - run:
           name: Run Web Tests
           command: |
-            lerna run test:web --stream --since -- -- --ci --bail --coverage
+            lerna run test:web --stream --since -- -- --ci --bail --coverage
```
Comment (karthickraja-ntsindia) [inline .circleci/config.yml:110]: extra space!!

File: packages/ssr/cypress.json [hunk 2]
```diff
@@ -6,5 +6,5 @@
   "screenshotsFolder": "__tests__/screenshots",
   "supportFile": "__tests__/cypress/support",
   "video": false,
-  "blockHosts": ["*spot.im"]
```
Comment (karthickraja-ntsindia) [inline packages/ssr/cypress.json:9]: need to remove ?

File: .circleci/config.yml [hunk 3]
```diff
@@ -147,8 +150,11 @@ jobs:
       - run:
           name: Run Cypress Tests
           command: |
-            yarn test:e2e
+            chmod +x ./lib/cypress-rerun.sh
```
[the remark this answers is not in the document]
Comment (Avi247h) [reply .circleci/config.yml]: done

File: packages/ssr/__tests__/integration/author-profile-default.js [hunk 4]
```diff
@@ -1,4 +1,14 @@
-import authorProfileTests from "../helpers/author-profile-helper";
+//import authorProfileTests from "../helpers/author-profile-helper";
 
 // default
-authorProfileTests({ stickyElements: ["#nav"], skipSnapshotTest: true });
+authorProfileTests({ stickyElements: ["nav"], skipSnapshotTest: true });
+const rerun = process.env.RERUN; // Access the environment variable
```
[the remark this answers is not in the document]
Comment (Avi247h) [reply packages/ssr/__tests__/integration/author-profile-default.js]: removed

File: packages/jest-configurator-web/src/jest-configurator.js [hunk 5]
```diff
@@ -34,6 +34,5 @@ export default (cwd, options = {}) => {
       "node_modules/(?!(@times-components|@storybook/react)/)"
     ]
   };
-
```
[the remark this answers is not in the document]
Comment (Avi247h) [reply packages/jest-configurator-web/src/jest-configurator.js:37]: removed

File: lib/cypress-rerun.sh [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+#  Team:        @tm-discovery-qa (Discovery)
+#
+
+
+
+echo "Creating directory structure at './packages/ssr/__tests__/test-results'..."
+mkdir -p "./packages/ssr/__tests__/test-results"
+ln -s /home/circleci/project/node_modules/cypress-circleci-reporter "/home/circleci/project/packages/ssr/node_modules/cypress-circleci-reporter"
+echo "Running yarn command: 'yarn test:e2e'..."
+yarn test:e2e:ci || error_exit "Yarn command failed."
```
[the remark this answers is not in the document]
Comment (Avi247h) [reply lib/cypress-rerun.sh]: Fixed and you reviewed it

File: .circleci/config.yml [hunk 7]
```diff
@@ -132,6 +133,8 @@ jobs:
 
   run_cypress_tests:
     executor: node
+    resource_class: xlarge
```
[the remark this answers is not in the document]
Comment (Avi247h) [reply .circleci/config.yml:136]: done

File: cypress.json [hunk 8]
```diff
@@ -1 +1 @@
-{}
+{ "reporter": "cypress-circleci-reporter"}
```
[the remark this answers is not in the document]
Comment (Avi247h) [reply cypress.json]: removed

File: package.json [hunk 9]
```diff
@@ -160,3 +162,4 @@
     "winston": "3.10.0"
   }
 }
+
```
[the remark this answers is not in the document]
Comment (Avi247h) [reply package.json:166]: done

File: packages/jest-configurator-web/src/jest-configurator.js [hunk 10]
```diff
@@ -34,6 +34,5 @@ export default (cwd, options = {}) => {
       "node_modules/(?!(@times-components|@storybook/react)/)"
     ]
   };
```
Comment (SaqueebNTS) [inline packages/jest-configurator-web/src/jest-configurator.js:36]: if this file is not changed then you should checkout for origin

**Review by SaqueebNTS — APPROVED** [review]

**Review by karthickraja-ntsindia — APPROVED** [review]

**Pull request merged**